html-aam/roles-contextual.html

```diff
@@ -40,6 +40,9 @@
   <header data-testname="el-header" aria-label="x" class="ex-generic">x</header>
 </nav>
 <header data-testname="el-header-ancestorbody" data-expectedrole="banner" class="ex">x</header>
+<main>
+  <header data-testname="el-header-ancestormain" data-expectedrole="generic" class="ex">x</header>
+</main>
 
```

Comment:
You need to use `class="ex-generic"` (without an `expectedrole` expando) or this will break Chromium/Gecko.

Reply:
FYI @joone @twilco

Reply:
See the generic header-inside-nav test that is immediately preceding this one.

Reply:
https://github.com/web-platform-tests/interop-accessibility/issues/48

Reply:
Done. Thanks for the review!